docs/standard/choosing-core-framework-server.md

```diff
@@ -1,108 +1,106 @@
 
 ### A need for side by side of .NET versions per application level
 
-If you want to be able to install applications with dependencies on different versions of frameworks in .NET, you need to use .NET Core, which provides 100% side-by-side. Easy side-by-side installation of different versions of .NET Core on the same machine allows you to have multiple services on the same server, each of them on its own version of .NET Core, eliminating risks and saving money in application upgrades and IT operations.
+If you want to be able to install applications with dependencies on different versions of .NET, we recommend you use .NET Core. .NET Core offers easy side-by-side installation of different versions of the .NET Core runtime on the same machine, allowing you to have multiple services on the same server, each of them on its own version of .NET Core, eliminating risks and saving money in application upgrades and IT operations.
 
 ## When to choose .NET Framework
 
-While .NET Core offers significant benefits for new applications and application patterns, the .NET Framework will continue to be the natural choice for many existing scenarios and as such, it won’t be replaced by .NET Core for all server applications.
+While .NET Core offers significant benefits for new applications and application patterns, the .NET Framework continues to be the natural choice for many existing scenarios and as such, it isn't replaced by .NET Core for all server applications.
 
 ### Current .NET Framework applications
 
-In most cases, you won’t need to migrate your existing applications to .NET Core. Instead, a recommended approach is to use .NET Core as you extend an existing application, such as writing a new web service in ASP.NET Core.
+In most cases, you don’t need to migrate your existing applications to .NET Core. Instead, a recommended approach is to use .NET Core as you extend an existing application, such as writing a new web service in ASP.NET Core.
 
 ### A need to use third-party .NET libraries or NuGet packages not available for .NET Core
 
-Libraries are quickly embracing .NET Standard, which enables sharing code across all .NET flavors including .NET Core. With .NET Standard 2.0 this will be even easier, as the .NET Core API surface will become significantly bigger and .NET Core applications can directly use existing .NET Framework libraries. This transition won’t be immediate, though, so we recommend checking the specific libraries required by your application before making a decision one way or another.
+Libraries are quickly embracing .NET Standard, which enables sharing code across all .NET flavors including .NET Core. With .NET Standard 2.0, this is even easier, as the .NET Core API surface became significantly larger and .NET Core applications can directly use existing .NET Framework libraries.
```

Comment:
Libraries are quickly embracing .NET Standard. .NET Standard enables sharing code across all .NET flavors including .NET Core. With .NET Standard 2.0, this is even easier, as the .NET Core API surface became significantly larger. .NET Core applications can directly use existing .NET Framework libraries.

That last sentence might need more connection. Sharring is made easier because .NET Core applications can directly use existing .NET Framework libraries.
flavors is slang - can we use a more precice word that will machine localize more easily? Variants?